CHANGELOG.md

```diff
@@ -15,6 +15,18 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 5.13.0
```

Comment:
Actually, this can be moved into the `5.12.0` section below since we haven't released it yet

Reply:
done